### ISSUE TITLE
Implement pipeline metrics

### ISSUE THREAD

**Issue by FedericoCeratto** [ooni/spec#143]
Implement application metrics (e.g. Prometheus, Statsd) for pipeline scripts and other scripts.
Requirements: collect data reliably from long and short-lived scripts.
Support real-time and long-term storage.
Optionally support gathering metrics during development (on desktops etc) without storing them in Prometheus.

**Comment by hellais** [comment]
Yes this is a good idea. Maybe @superq has some tips on how to best do in this prometheus especially the long term storage aspect.
